closed/make/common/Modules.gmk

```diff
@@ -36,6 +36,17 @@ TOP_SRC_DIRS += \
     $(SUPPORT_OUTPUTDIR)/j9jcl_sources \
     #
 
+# openj9 specific modules for which javadoc is required
+DOCS_MODULES += \
+    openj9.cuda \
+    openj9.dataaccess \
+    openj9.dtfj \
+    openj9.gpu \
+    openj9.jvm \
+    openj9.sharedclasses \
+    openj9.traceformat \
+    #
```

Comment:
The module openj9.dtfjview is intended to be used only by openj9.dtfj so javadoc is not required to be produced for it. This seemed to at least make it obvious which modules we are generating javadoc for.

Reply:
An explicit list means it will need to be updated when modules are added or removed.

The relationship between openj9.dtfj and openj9.dtfjview is the other way around: the latter requires the former. What's the problem with including javadoc for openj9.dtfjview?

Reply:
I don't think it is a problem to include openj9.dtfjview. The javadoc also includes other modules that don't contain content, such as jdk.jcmd, jdk.rmic. I've added a module comment, so we shouldn't get the copyright notice as the description any more.

Reply:
OK, I'll include all the openj9.xxx modules then.

Reply:
Changed to use a function.  Note that I've moved the openj9 javadoc generation code to a new Docs-post.gmk file so as not to move the original closed/make/Docs.gmk include in make/Docs.gmk (and added a new include for closed/make/Docs-post.gmk).